client/post-editor/editor-feedback-invitation/index.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { identity } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+function EditorFeedbackInvitation( { translate, onTrigger } ) {
+	return (
+		<div className="editor-feedback-invitation">
+			<p className="editor-feedback-invitation__invitation">
+				{ translate(
+					'{{highlight}}Need another set of eyes?{{/highlight}} Ask a friend to review your post.',
+					{
+						components: {
+							highlight: <span className="editor-feedback-invitation__highlight" />
+						}
+					}
+				) }
+			</p>
+			<Button className="editor-feedback-invitation__button" onClick={ onTrigger }>
+				{ translate( 'Send to a Friend' ) }
+			</Button>
+		</div>
+	);
+}
```

Comment:
Interesting! I know I've been frustrated by the issue with anonymous functions and assumed it would be an issue with arrow functions as well. I'm guessing that the devtools have improved to address this pain, which is cool. Thanks for showing me this.

I no longer have a reason to avoid arrow functions for components, except that I like reading `function SomeHandyComponent() {}`. :)